docs/deprecated-work/bri/overview-of-reference-implementations.md

```diff
@@ -23,9 +23,9 @@ Companies and individuals contributing to the Baseline Protocol are not putting
 
 To this end, Reference Implementations -- not unlike different implementations of the Ethereum Yellow Paper -- may build-in dependencies on specific products or add proprietary components and tools that might feature or advantage a company or group of companies. This is allowed -- and encouraged -- so long as **the Reference Implementation does not introduce confusing naming or positioning that would give a developer the sense that those elements are essential for **_**baselining**_. That said, the best Reference Implementations will endeavor to be modular so that their work can be used with a variety of components without someone having to perform "surgery" on the code.
 
-Over time, it is expected that many implementations -- both proprietary and otherwise -- will be developed and not submitted back into the Baseline Protocol open source repository. But the community is grateful to those companies and individuals that \_provide \_their work as contributions back to open source. These contributions are stored in the [examples/ ](https://github.com/ethereum-oasis/baseline/tree/master/examples)folder of the repository under the naming convention below.
+Over time, it is expected that many implementations -- both proprietary and otherwise -- will be developed and not submitted back into the Baseline Protocol open source repository. But the community is grateful to those companies and individuals that \_provide \_their work as contributions back to open source. These contributions are stored in the [examples/ ](https://github.com/ethereum-oasis-op/baseline/tree/master/examples)folder of the repository under the naming convention below.
```

Comment:
```suggestion
Over time, it is expected that many implementations -- both proprietary and otherwise -- will be developed and not submitted back into the Baseline Protocol open source repository. But the community is grateful to those companies and individuals that \_provide \_their work as contributions back to open source. These contributions are stored in the [examples/ ](https://github.com/ethereum-oasis-op/baseline/tree/main/examples)folder of the repository under the naming convention below.
```